Demos/Basic-Swift/Basic-Swift/SampleCollectionViewController.swift

```diff
@@ -52,64 +38,60 @@ class SampleCollectionViewController: UICollectionViewController {
     let validateButtonItem = UIBarButtonItem(title: "Validate",
       style: .Done,
       target: self,
-      action: NSSelectorFromString("validateButtonAction"))
+      action: "validateButtonAction")
 
     let updateButtonItem = UIBarButtonItem(title: "Update",
       style: .Done,
       target: self,
-      action: NSSelectorFromString("updateButtonAction"))
+      action: "updateButtonAction")
 
-    let flexibleBarButtonItem = UIBarButtonItem(barButtonSystemItem: .FlexibleSpace,
-      target: nil,
-      action: nil)
+    let flexibleBarButtonItem = UIBarButtonItem(barButtonSystemItem: .FlexibleSpace)
 
-    let readOnlyView = UIView(frame: CGRectMake(0.0, 0.0, 150.0, 40.0))
-    let readOnlyLabel = UILabel(frame: CGRectMake(0.0, 0.0, 90.0, 40.0))
+    let readOnlyView = UIView(frame: CGRectMake(0, 0, 150, 40))
+    let readOnlyLabel = UILabel(frame: CGRectMake(0, 0, 90, 40))
     readOnlyLabel.text = "Read-Only"
     readOnlyLabel.textColor = UIColor(fromHex: "5182AF")
-    readOnlyLabel.font = UIFont.boldSystemFontOfSize(17.0)
+    readOnlyLabel.font = UIFont.boldSystemFontOfSize(17)
```

Comment:
it's ok for me to use 17.0 , and .0 for all float and double numbers. 

But i feel in swift it's more about cleaner and removing distraction 
you say what you want not ho: 
I want font of size 17 
I want a font of size with Float number 17.0.
P.S the same is valid for using `self`